### PR TITLE
Integration test, travis quick-fix

### PR DESCRIPTION
**Pull request by bryanchriswhite**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -74,7 +74,7 @@ matrix:
         - make test-captplanet
 
     ### certificate signing tests ###
-    - env: MODE=integration
```
Comment (egonelbre) [inline .travis.yml]: Let's merge these tests together with captplanet tests:
```
    ### Integration tests ###
    - env: MODE=integration
      services:
        - redis
      install:
        - source scripts/install-awscli.sh
        - go install -race storj.io/storj/cmd/{captplanet,storj-sdk,storagenode,satellite,uplink,gateway}
      script:
        - make test-captplanet
        - make test-certificate-signing
```

Comment (egonelbre) [reply .travis.yml]: Also ensure that the `test-certificate-signing` is built using `-race`

Comment (bryanchriswhite) [reply .travis.yml]: No objection here but do you care to explain why?

Comment (egonelbre) [reply .travis.yml]: Time needing to wait before somebody can merge something. Every travis job eats up the resources meaning with multiple concurrent PR-s building we need to wait that much longer.

**Pull request merged**